types/decimal.go

```diff
@@ -241,6 +241,16 @@ func (d Dec) Quo(d2 Dec) Dec {
 	return Dec{chopped}
 }
 
+// quotient
+func (d Dec) QuoInt(i Int) Dec {
+	mul := new(big.Int).Quo(d.Int, i.i)
+
+	if mul.BitLen() > 255+DecimalPrecisionBits {
```

Comment:
This check isn't needed, since quotienting can't increase the size of the number.